operator/pkg/apis/operator/v1alpha1/type.go

```diff
@@ -236,22 +236,33 @@ type VolumeData struct {
 }
 
 // ExternalEtcd describes an external etcd cluster.
-// operator has no knowledge of where certificate files live, and they must be supplied.
+// Operator has no knowledge of where certificate files live, and they must be supplied.
 type ExternalEtcd struct {
 	// Endpoints of etcd members. Required for ExternalEtcd.
 	Endpoints []string `json:"endpoints"`
 
 	// CAData is an SSL Certificate Authority file used to secure etcd communication.
 	// Required if using a TLS connection.
-	CAData []byte `json:"caData"`
+	// This field is deprecated and will be removed in a future version. Use SecretRef for providing client connection credentials.
```

Comment:
```suggestion
	// Deprecated: This field is deprecated and will be removed in a future version. Use SecretRef for providing client connection credentials.
```
Let's give it a deprecation tag. 

Same as below